### ISSUE TITLE
Support notification of READ events

### ISSUE THREAD

**Issue by cannon73** [fsnotify/fsnotify#346]
I have some doubts about the fs-notify software. There is no way to notify file reads but the i-notify software supports it through unix.IN_ACCESS. Kindly add support for notify read events.

**Comment by arp242** [comment]
https://github.com/fsnotify/fsnotify/issues/352 should be done first, as many people don't care about read events. Also need to look in to cross-platform support.

**Comment by arp242** [comment]
I created a new issue to track adding additional events: #519 Bit easier to have one issue instead of a bunch for every possible event.
